src/ngx_http_lua_ssl_session_fetchby.c

```diff
@@ -442,15 +442,18 @@ ngx_http_lua_log_ssl_sess_fetch_error(ngx_log_t *log, u_char *buf, size_t len)
 
     c = log->data;
 
-    if (c->addr_text.len) {
-        p = ngx_snprintf(buf, len, ", client: %V", &c->addr_text);
-        len -= p - buf;
-        buf = p;
-    }
+    if (c) {
```

Comment:
The same comment with the https://github.com/openresty/lua-nginx-module/pull/1845#pullrequestreview-564921967

Reply:
fixed